presto-docs/src/main/sphinx/functions/ml.rst

```diff
@@ -0,0 +1,151 @@
+===================
+Machine Learning Functions
+===================
+
+The machine learning plugin in Presto provides machine learning functionality
+as an aggregation function. It enables us to train Support Vector Machine (SVM)
+based classifier and regressor for the supervised learning problems.
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+.. note::
+  These machine learning functions are not fully optimized in a distributed manner.
+  Since the final training process is executed in a single instance, it has a limited capability to train the enormous scale data for now.
+
+Feature Vector
+--------------
+
+To solve a problem with the machine learning technique especially as a
+supervised learning problem, it is necessary to represent the data set
+with the sequence of pairs of labels and feature vector. A label is a
+target value you want to predict from the unseen feature and a feature is a
+An N-dimensional vector whose elements are numerical values. In Presto, feature
+vector is represented as a map type value whose key is an index of each
```

Comment:
In Presto, a feature vector is represented as a map-type value, whose key is an index of each feature, so that it can express a sparse vector.